t/admin/upstream3.t

```diff
@@ -91,5 +91,3 @@ GET /t
 --- error_code: 400
 --- response_body
 {"error_msg":"invalid configuration: property \"retry_timeout\" validation failed: expected -1 to be greater than 0"}
---- no_error_log
```

Comment:
Why remove this?

Reply:
https://github.com/apache/apisix/blob/42ab6149272920bbab66985b90e2754a7c6cb747/t/admin/upstream3.t#L32-L34
In file `upstream3.t` has inserted this block when no error